services/core/fuse/web-client/src/grpc/registry/key_value/v1/service_pb.js

```diff
@@ -0,0 +1,106 @@
+// @generated by protoc-gen-es v1.6.0 with parameter "target=js"
```

Comment:
Can we have the web-client reference the generated JS/TS code under `api/` instead of having a copy here?

Reply:
I think we can. TBH I just copied what was in `blueprint` to get CI working. I'm going to nuke everything and put a barebones web-client in.